server/types/schemas.ts

```diff
@@ -1,10 +1,11 @@
-import { ObjectId } from '../deps.ts';
+import { ObjectId } from "../deps.ts";
 
 export interface RoomSchema {
   _id: ObjectId;
   roomCode: string;
   moderatorId: ObjectId;
   options: number[];
+  voters: string[];
```

Comment:
Should this be an array of ObjectIds? Like the ObjectIds returned from creating the users in the user table? If we just rely on names, we may not be able to tell two people apart who have the same name.

Reply:
yeah it should be.  Good call.